system-int-test/test/cmr/system_int_test/ingest/provider_ingest_permissions_test.clj

```diff
@@ -223,6 +226,59 @@
            another-prov-admin-token
            provider-admin-read-token))
 
+    (testing "ingest subscription update permissions"
+      (are3 [token]
+            (ingest-succeeded?
+              (ingest/ingest-concept subscription {:token token
+                                                   :allow-failure? true}))
```

Comment:
are there test cases you could add with `:allow-failure? false`?

Reply:
allow-failure? false will cause exception to be thrown when ingest fails and we can't check errors, status etc... I think all our tests have "true" case.